tests/support/cluster_util.tcl

```diff
@@ -277,6 +277,14 @@ proc cluster_get_myself id {
     return {}
 }
 
+# Returns the parsed "myself's primary" node entry as a dictionary.
```

Comment:
I guess all of these functions are about CLUSTER NODES? It can be useful to mention it.

```suggestion
# Returns the parsed "myself's primary" CLUSTER NODES entry as a dictionary.
```

Reply:
yes